packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -212,4 +213,24 @@ export function BindStateToController(
   };
 }
 
+export function AriaRegion(regionName: string) {
```

Comment:
Maybe we can create a new accessibility-utils ?

Reply:
Additionally, I changed my mind and I'll rename `AriaRegion` to `AriaLiveRegion`.